assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -40,15 +43,15 @@ export class CampaignSidebar extends Component {
   // When theme switches, we need to check if any options were previously chosen that are not allowed in the new theme.
   // For each of these, we either set them to the default value
   async handleThemeSwitch( metaKey, value, meta ) {
-    const theme = await loadTheme( value )
-    const prevTheme = this.state.theme;
-    this.setState({ theme });
+    const newTheme = await loadTheme( value )
```

Comment:
I just renamed a few things to make it a bit more understandable. If you look at what this function does in detail, it's definitely not ideal. For example having it loop the meta instead of the theme would make more sense. But I'd rather not modify the logic unless it's really needed to avoid refactor hell :sweat_smile: